src/ashell/ide-webusb.c

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2017-2018, Intel Corporation.
+
+/**
+* @file
+* @brief WebUSB specific configuration file
+*
+* Provides WebUSB and Microsoft OS Descriptor 2.0 Platform Capability
+* descriptors. Also, it handles the WebUSB Device Requests and provides
+* the landing page and allowed origin descriptors.
+*/
+
+#include <stdio.h>
+#include <string.h>
+#include <device.h>
```

Comment:
split into to C includes and Zephyr includes

Reply:
Yeah, I like to first include C lib stuff, then include Zephyr stuff, then finally include ZJS stuff... i.e. most standard to least standard. FWIW.

Reply:
Done.